db/seeds.rb

```diff
@@ -11,3 +11,10 @@
 Hall.create(name: 'Green Hall', capacity: 200)
 Hall.create(name: 'Second Hall', capacity: 400)
 Hall.create(name: 'Oh no Hall', capacity: 100)
+
+Movie.create(title: 'First Movie', duration: '120', adult_only: 0, description: 'Some description of the first movie',
```

Comment:
you created column `adult_only` as `boolean` and here you have `integer`

Reply:
ok I didn't think about it, but I can see that my render is in "true" "false" but seeds "0" and "1". 
![image](https://user-images.githubusercontent.com/84372602/120295411-67c24f00-c2c7-11eb-9004-78791ef17ab3.png) 
![image](https://user-images.githubusercontent.com/84372602/120295449-6c870300-c2c7-11eb-8766-5b8b097592f5.png)

Reply:
Please check https://api.rubyonrails.org/classes/ActiveModel/Type/Boolean.html.